src/main/resources/lang/de_de.json

```diff
@@ -0,0 +1,121 @@
+{
+	"language": "Deutsch",
+
+	"mapping_format.enigma_file": "Enigma‐Datei",
+	"mapping_format.enigma_directory": "Enigma‐Verzeichnis",
+	"mapping_format.enigma_zip": "Enigma‐ZIP‐Archiv",
+	"mapping_format.tiny_v2": "Tiny‐v2‐Datei",
+	"mapping_format.tiny_file": "Tiny‐Datei",
+	"mapping_format.srg_file": "SRG‐Datei",
+	"mapping_format.proguard": "ProGuard‐Datei",
+	"type.methods": "Methoden",
+	"type.fields": "Felder",
+	"type.parameters": "Parameter",
+	"type.classes": "Klassen",
+
+	"menu.file": "Datei",
+	"menu.file.jar.open": "Java‐Archiv öffnen …",
+	"menu.file.jar.close": "Java‐Archiv schließen",
+	"menu.file.mappings.open": "Zuordnungen laden …",
+	"menu.file.mappings.save": "Zuordnungen speichern",
+	"menu.file.mappings.save_as": "Zuordnungen speichern unter …",
+	"menu.file.mappings.close": "Zuordnungen schließen",
+	"menu.file.mappings.drop": "Ungültige Zuordnungen löschen",
+	"menu.file.export.source": "Quelldateien exportieren …",
+	"menu.file.export.jar": "Java‐Archiv exportieren …",
+	"menu.file.stats": "Zuordnungsstatistiken …",
+	"menu.file.stats.title": "Enthaltene Elemente auswählen",
+	"menu.file.stats.generate": "Statistiken generieren",
+	"menu.file.exit": "Beenden",
+	"menu.decompiler": "Dekompilierer",
+	"menu.view": "Ansicht",
+	"menu.view.themes": "Farbgestaltung",
+	"menu.view.themes.default": "Standard",
+	"menu.view.themes.darcula": "Darcula",
+	"menu.view.themes.system": "System",
+	"menu.view.themes.none": "Keine (JVM‐Standard)",
+	"menu.view.languages": "Sprache",
+	"menu.view.languages.title": "Sprache ändern",
+	"menu.view.languages.summary": "Die ausgewählte Sprache wird nach dem nächsten Neustart verwendet.",
+	"menu.view.languages.ok": "Okay",
+	"menu.view.search": "Suche",
+	"menu.help": "Hilfe",
+	"menu.help.about": "Über",
+	"menu.help.about.title": "%s – Informationen",
+	"menu.help.about.ok": "Ok",
+	"menu.help.github": "GitHub‐Projekt",
+
+	"popup_menu.rename": "Umbenennen",
+	"popup_menu.javadoc": "Javadoc bearbeiten",
+	"popup_menu.inheritance": "Vererbung zeigen",
```

Comment:
Why use 'zeigen' and not 'anzeigen' here unlike everywhere else?

Reply:
Thank you very much, the consistency has now been tightened.